internal/datastore/mysql/migrations/zz_migration.0005_extend_object_id.go

```diff
@@ -0,0 +1,22 @@
+package migrations
+
+import "fmt"
+
+// Extend the object ID column to 1024 characters
+func adjustObjectIDLength(t *tables) string {
+	return fmt.Sprintf(
+		`ALTER TABLE %s
+		CONVERT TO CHARACTER SET latin1,
```

Comment:
we should make absolutely sure that the IDs and the names of objects and relations are within latin1 in the authzed-go unit tests, once we start making changes there